### ISSUE TITLE
TypeError: Object.keys called on non-object

### ISSUE THREAD

**Issue by bmvakili** [dowjones/gulp-bundle-assets#88]
Hello, with new release, getting error:

```
/opt/my_project/node_modules/gulp-bundle-assets/lib/results/index.js:54
            _.each(Object.keys(result.contents[key]).sort(), function(typeKey)
                          ^
TypeError: Object.keys called on non-object
  at Function.keys (native)
  at /opt/my_project/node_modules/gulp-bundle-assets/lib/results/index.js:54:27
  at arrayEach (/opt/my_project/node_modules/gulp-bundle-assets/node_modules/lodash/index.js:1314:13)
  at Function.<anonymous> (/opt/my_project/node_modules/gulp-bundle-assets/node_modules/lodash/index.js:3355:13)
  at /opt/my_project/node_modules/gulp-bundle-assets/lib/results/index.js:51:13
  at /opt/my_project/node_modules/gulp-bundle-assets/node_modules/lodash/index.js:3106:15
  at baseForOwn (/opt/my_project/node_modules/gulp-bundle-assets/node_modules/lodash/index.js:2053:14)
  at /opt/my_project/node_modules/gulp-bundle-assets/node_modules/lodash/index.js:3076:18
  at Function.<anonymous> (/opt/my_project/node_modules/gulp-bundle-assets/node_modules/lodash/index.js:3356:13)
  at /opt/my_project/node_modules/gulp-bundle-assets/lib/results/index.js:48:11
  at /opt/my_project/node_modules/gulp-bundle-assets/node_modules/mkdirp/index.js:48:26
  at Object.oncomplete (fs.js:107:15)
```

Above's result of `gulp buildAll` command.

The said line was recently changed here:
https://github.com/dowjones/gulp-bundle-assets/commit/30afbc87ec3855d38ad4a80dede5d54423dd3701#diff-cc30f4d57e32ddd61eb96d387c60c412R53

As workaround, I've wrapped in a try-catch:

```
try {
            _.each(Object.keys(result.contents[key]).sort(), function(typeKey) {
              sortedBundleTypes[typeKey] = result.contents[key][typeKey];
            });
} catch (err) {}
```

For clarity, version details below:
```
/opt/my_project/node_modules/gulp-bundle-assets/package.json:

{
"name": "gulp-bundle-assets",
"version": "2.27.0"
...
```

**Comment by chmontgomery** [comment]
Sorry, this was me. I will get a fix out tonight for you

**Comment by chmontgomery** [comment]
@bmvakili a fix has been published to `v2.27.1`. Can you please test and make sure it works for you? Thanks!

**Comment by bmvakili** [comment]
Our cache just let me test this new release; and confirming build's finishing again. Thanks @chmontgomery for quick fix!
